CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 6.1-0.12.11 (2019-11-10)
```

Comment:
Please remove changes to this file

Reply:
changes removed 👍

Reply:
didn't expect you to respond so quickly so I took care of it.  😛 

sorry I took so long.  Thanks again for the PR!